ui/js/treeherder.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+/*exported treeherder*/
 var treeherder = angular.module('treeherder',
```

Comment:
I feel this comment needs to be there, otherwise treeherder will throw no-unused-var as it is not being used in the same file.